src/components/desktop/SidePane.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components/macro'
+
+import Search from '../search/Search'
+
+const PaneContainer = styled.div`
```

Comment:
Do we need a while styled component for this? Its just a wrapper, but maybe its the cleaner solution.